builtin/credential/ldap/path_config.go

```diff
@@ -143,8 +166,22 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	groupfilter := d.Get("groupfilter").(string)
```

Comment:
I didn't realize we had that. I think that could work, unless there is a reason that you might want to specify an empty groupfilter -- is there?